tests/python/VultureBearTest.py

```diff
@@ -42,85 +52,64 @@ def prep_file():
 
             return list(self.uut.run())
 
+    def verify_results(self, test_case, expected):
+        detected = dict((item.message, (item.affected_code[0].start.line,
+                                        item.affected_code[0].end.line,
+                                        item.confidence))
+                        for item in self.get_results(test_case))
+        self.assertEqual(detected, expected)
+
     def test_used_variable(self):
-        good_file = """
-        x = 2
-        print(x)
-        """
-        self.assertEqual(len(self.get_results(good_file)), 0)
+        self.verify_results(load_testfile('used_variable.py'), {})
 
     def test_unused_variable(self):
-        bad_file = """
-        b = 10
-        a = 12
-        print(a)
-        """
-        self.assertEqual(len(self.get_results(bad_file)), 1)
-
-    def test_unused_parameter(self):
-        bad_file = """
-        def test(a):
-            return 1
-
-        test(1)
-        """
-        self.assertEqual(len(self.get_results(bad_file)), 1)
-
-    def test_unused_function(self):
```

Comment:
also for this one I can't find the corresponding test file :+1:

Reply:
Same here, I changed it a little bit :p 
Here: https://github.com/coala/coala-bears/pull/2000/files#diff-3e91e594a33cd00f926e16490633fadcR1